eth/protocols/eth/handlers.go

```diff
@@ -269,6 +278,22 @@ func handleGetOnePendingEtxs66(backend Backend, msg Decoder, peer *Peer) error {
 	return peer.SendPendingEtxs(*pendingEtxs)
 }
 
+func handleGetOnePendingEtxsRollup66(backend Backend, msg Decoder, peer *Peer) error {
+	// Decode the block pending etxs rollup retrieval message
+	var query GetOnePendingEtxsRollupPacket66
+	if err := msg.Decode(&query); err != nil {
+		return fmt.Errorf("%w: message %v: %v", errDecode, msg, err)
+	}
+	requestTracker.Fulfil(peer.id, peer.version, GetOnePendingEtxsRollupMsg, query.RequestId)
+	pendingEtxs := backend.Core().GetPendingEtxsRollup(query.Hash)
+	if pendingEtxs == nil {
+		log.Debug("Couldn't complete a pendingEtxs request for", "Hash", query.Hash)
+		return nil
```

Comment:
It's a design choice, but we could choose to return an error here depending on how severe it is. Same for previous function.

Reply:
Returning error has negative effect on the peer relation, so I think in this case, no need to return error